Adds support for Portage's "getdelta" incremental update client.

Also disables FEATURES option "parallel-fetch" because slow
connections which use "getdelta" usually also prefetch sources
before starting to recompile, as bandwidth allocation has to
be monitored manually in order to avoid race conditions with
other downloads.

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -198,7 +198,7 @@ source /usr/portage/local/layman/make.co
 # will be available.
 #
 # Default fetch command (5 tries, passive ftp for firewall compatibility)
-#FETCHCOMMAND="/usr/bin/wget -t 5 -T 60 --passive-ftp \${URI} -P \${DISTDIR}"
+FETCHCOMMAND="/usr/bin/getdelta.sh \${URI} \# \${FILE}"
 #RESUMECOMMAND="/usr/bin/wget -c -t 5 -T 60 --passive-ftp \${URI} -P \${DISTDIR}"
 #
 # Using wget, ratelimiting downloads
